frontend/src/components/lobby/create-lobby/Lobby.jsx

```diff
@@ -60,6 +60,8 @@ export default function App() {
 
     const handleBeforeUnload = useCallback((event) => {
         event.preventDefault();
+        console.log('lobbyData', JSON.stringify(lobbyData))
+        sessionStorage.setItem("lobbyData", JSON.stringify(lobbyData));
```

Comment:
Így már a nem leadereknek is is van tárolva  sessionStorage-ban a lobby configja